README.md

```diff
@@ -1,25 +1,29 @@
-# GitHub Actions for Fermyon Cloud/Spin apps
+# `fermyon/actions` - GitHub Action collection for Spin and Fermyon Cloud
 
-The `fermyon/actions` is a collection of Github actions to enable running [spin](https://github.com/fermyon/spin) commands in your GitHub Actions workflow.
 
-It enables the following usecases:
+With the `fermyon/actions` collection, you can incorporate [Spin](https://developer.fermyon.com/spin/index) and [Fermyon Cloud](https://developer.fermyon.com/cloud/index) in your [GitHub Action](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow). 
 
-- setup spin and plugins (`fermyon/actions/spin/setup`)
-- build and push to OCI registry (`fermyon/actions/spin/push`)
-- deploy to Fermyon Cloud (`fermyon/actions/spin/deploy`)
-- deploy PR preview to Fermyon Cloud (`fermyon/actions/spin/preview`)
+This collection of Actions enables the following use cases:
 
-## `fermyon/action/spin/setup`
+- [x] set up Spin CLI and plugins using [`fermyon/actions/spin/setup@v1`](#install-spin-cli-and-plugins---fermyonactionsspinsetupv1)
+- [x] build and push your Spin app to an OCI registry using [`fermyon/actions/spin/push@v1`](#push-spin-app-to-an-oci-registry---fermyonactionsspinpushv1)
+- [x] deploy your Spin app to Fermyon Cloud using [`fermyon/actions/spin/deploy@v1`](#deploy-spin-app-to-fermyon-cloud---fermyonactionsspindeployv1)
+- [x] deploy PR preview to Fermyon Cloud [`fermyon/action/spin/preview@v1`](#deploy-preview-of-spin-app-to-fermyon-cloud---fermyon/action/spin/preview@v1)
```

Comment:
In the other links, the `/` and `@` characters are slugified away - is it correct that that is not done in this one?